aten/src/ATen/fbgemm_utils.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#ifdef USE_FBGEMM
+#include "fbgemm/Fbgemm.h"
+#include "fbgemm/QuantUtils.h"
+
+// The struct for the packed weight matrix (PackBMatrix) and the corresponding
+// column offsets used for the fully connect layer, which are both prepared in
+// the prepacking step to save the computations in the inference. Note the
+// column offsets includes the sum of the B columns as well as the scalar term
```

Comment:
Maybe worth mentioning column offset is needed for asymmetric quantization of input (affine quantization in PT).

Reply:
Will add these comments.